app/linux-prebuild.sh

```diff
@@ -1,7 +1,25 @@
 #!/bin/bash
+
 set -e # Quit script on error
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-echo "Warning: Unix build scripts are still a work in progress!"
+
+echo "Installing vcpkg"
```

Comment:
Will we be updating this so that the message only appears if vcpkg is not already installed? (Same for other platforms)?

Reply:
Sure, that's simple enough; that's the point of these draft PRs; to give people a chance to comment on work in progress :)
This step is designed to make the prebuild re-run quick if you have already done it once.